src/argus_htmx/incidents/views.py

```diff
@@ -208,8 +218,9 @@ def incident_list(request: HtmxHttpRequest) -> HttpResponse:
         "base": base_template,
         "page": page,
         "last_refreshed": last_refreshed,
-        "update_interval": 30,
-        "per_page": PAGE_SIZE,
+        "update_interval": 3000,
```

Comment:
Why do you change `update_interval`? (Should probably also be a setting, though.)

Reply:
haha oops, that's a leftover from my development flow. I often change it to a very high value so that I can properly inspect the DOM without interruptions (one reason why I'd like the update interval configurable)

Reply:
anyway I've reverted it